Sources/SwerverCore/Router/Router.swift

```diff
@@ -33,11 +34,13 @@ public class Router {
         return responseHeaderFormatter.format(httpResponse: notFoundAction.dispatch())
     }
 
-    private func isOptionsRequest(request: HttpRequest) -> Bool {
-        return request.getMethod() == RequestMethod.options
+    private func isOptionsRequest(request: HttpRequest?) -> Bool {
```

Comment:
In theory, this should never be called if the request is nil, therefore you can probably keep this as `HttpRequest` instead of `HttpRequest?`. This is related to my other comment about unwrapping `request`

Reply:
this function is actually removed in the feature-refactoring branch as it's no longer needed